src/Canvas.h

```diff
@@ -0,0 +1,26 @@
+#ifndef __CANVAS_H_INCLUDED__
+#define __CANVAS_H_INCLUDED__
+
+
+#include <iostream>
+#include <fstream>
+
+using namespace std;
+typedef char byte;
+
+struct RGB {
+	byte red;
+	byte green;
+	byte blue;
+};
+
+
+__interface ICanvas {
+	void init(const char* fileName, int size); // might be private if you invoke it from constructor
+	void drawPixel(const char* fileName, uint16_t x, uint16_t y, RGB color, int size);
```

Comment:
The `size` parameter is pretty confusing for `drawPixel()` method.  The common meaning of the pixel is 1x1 atomic piece of image. No size is implied.
If you want to draw _a line_ (which has size), please provide another method for that.